frontend/src/views/Bridge.vue

```diff
@@ -193,15 +232,15 @@
                   {{$t('bridge.fromChain')}}: {{supportedChains[supportedChainIds.indexOf(weapon['2'])]}}
                 </option>
               </select>
-              <div class="p-2 text-center">
+              <div class="mt-2 text-center">
                 <b-button :disabled="!weaponIdToWithdraw" variant="primary" @click="withdrawBridge(weaponIdToWithdraw)"
-                          class="gtag-link-others" tagname="click_transfer_bridge"> {{ $t('bridge.withdrawWeapon') }}
+                          class="gtag-link-others withdrawBtn" tagname="click_transfer_bridge"> {{ $t('bridge.withdrawWeapon') }}
                 </b-button>
               </div>
             </div>
             <div v-if="incomingChars.length !== 0" class="p-4 w-20">
-              <h4 class="text-center">{{$t('bridge.selectCharacterToWithdraw')}}</h4>
-              <select class="form-control" v-model="characterIdToWithdraw">
+              <h4 class="text-center withdrawText">{{$t('bridge.selectCharacterToWithdraw')}}</h4>
+              <select class="form-control withdrawSelect" v-model="characterIdToWithdraw">
                 <option value="" disabled selected>{{$t('bridge.selectYourCharacter')}}</option>
                 <option v-for="character in incomingChars"
                 :value="character['7']"
```

Comment:
If there was enough time for that, I would refactor the way we are handling types in a bridge, character should be a type here, we should access a particular attribute, not access it via '7', similar in other places in code here